rest_framework/__init__.py

```diff
@@ -1,4 +1,4 @@
-"""
+r"""
```

Comment:
This was intentional. Strings now raise an "invalid escape sequence" deprecation warning ([issue](https://bugs.python.org/issue27364)). Using a raw string gets around this.  I tried using comments, but it looked kind of funny.